plugins/rest/rest_auth.go

```diff
@@ -29,6 +29,11 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var (
```

Comment:
[nit]
```suggestion
const (
```
Ah wait we can't do that if we want to take its address... Fine. 😄

Reply:
Hehe yeah I started with const and quickly learnt that quirk of Go! :)

Reply:
If we really wanted to, we could do something like
```go
const def = "s3"

// in some func
s := def
&s // use that reference
```

I guess, but this is OK.

Reply:
Good shout I'll do that as while a bit more roundabout it's nice to have the constant be a bit more constant :D